src/common/libschedutil/init.h

```diff
@@ -21,6 +21,10 @@ extern "C" {
 
 typedef struct schedutil_ctx schedutil_t;
 
+enum schedutil_flags {
+    SCHEDUTIL_FREE_NOLOOKUP = 1, // ops->free() will be called with R=NULL
```

Comment:
Looking good.  Would probably be a good idea to comment in ops.h that 'R' is NULL if this flag was passed to `schedutil_create()`, and that one should do so if R is not needed since it saves a KVS lookup.